conda_build/index.py

```diff
@@ -781,12 +781,20 @@ def __init__(self, channel_root, channel_name, subdirs=None, threads=MAX_THREADS
         self.deep_integrity_check = deep_integrity_check
 
     def index(self, patch_generator, hotfix_source_repo=None, verbose=False, progress=False,
-              current_index_versions=None, index_file=None):
+              current_index_versions=None, index_file=None, append=None):
         if verbose:
             level = logging.DEBUG
         else:
             level = logging.ERROR
 
+        # dict(subdir: repodata)
+        existing_repodatas = {}
+        append = [] if not append else append
+        for repodata in append:
+            with open(repodata) as fh:
+                j = json.load(fh)
+            existing_repodatas[j.get("info", {}).get("subdir", None)] = j
```

Comment:
```suggestion
        append = append or []
        for repodata in append:
            with open(repodata) as repodata_file:
                loaded = json.load(repodata_file)
            repodata_subdir = loaded.get("info", {}).get("subdir", None)
            existing_repodatas[repodata_subdir] = loaded_repodata
```
Just a bit easier on the eyes.